packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -14,39 +15,51 @@ const BreadcrumbSeparator: React.FC = () => (
     paddingLeft="space20"
     paddingRight="space20"
     role="presentation"
+    element={`${element}_SEPARATOR`}
   >
     /
   </Text>
 );
 
 export interface BreadcrumbItemProps extends React.HTMLAttributes<HTMLLIElement> {
   children: NonNullable<React.ReactNode>;
+  element?: BoxElementProps['element'];
   href?: string;
   last?: boolean;
 }
 
 const BreadcrumbItem = React.forwardRef<HTMLAnchorElement, BreadcrumbItemProps>(
-  ({children, href, last, ...props}, ref) => {
+  ({children, element, href, last, ...props}, ref) => {
+    const separatorElementName = element || 'BREADCRUMB';
+    const elementName = element || 'BREADCRUMB_ITEM';
```

Comment:
The same behavior can be achieved more succinctly by doing `element = 'BREADCRUMB_ITEM',` on line 32

Reply:
I intentionally did not use a default here since so I could conditionally pass the consumer provided element string, if it is defined, and if not use `BREADCRUMB_SEPARATOR` and `BREADCRUMB_ITEM` as defaults for the breadcrumb item and the separator component respectively.
Notice that the separator is named `BreadcrumbSeparator` and we have been using default names that match the component name.

I could have alternatively added some conditional logic like:
```
<BreadcrumbSeparator element={element === 'BREADCRUMB_ITEM' ? 'BREADCRUMB_SEPARATOR' : `${element}_SEPARATOR`}
```

I chose not to do that though, since inline ternary operators like this are less readable.

Reply:
if `element="BREADCRUMB_ITEM"`, then the separator will be `BREADCRUMB_ITEM_SEPARATOR` which is fine.
if the user passes and element (`element="FOO"`), then the separator will be `FOO_SEPARATOR` which is also fine.

This is how we handle it everywhere

Reply:
Oh but the separator is called `BreadcrumbSeparator`. I think yesterday in focus time we said that it should not include `Item`. If that's not a requirement, I'm happy to change.